modules/tinymce/src/themes/silver/test/ts/headless/components/urlinput/UrlInputTest.ts

```diff
@@ -160,7 +159,7 @@ describe('headless.tinymce.themes.silver.components.urlinput.UrlInputTest', () =
             classes: [ arr.has('tox-collection__group') ],
             children: [
               s.element('div', {
-                classes: [ arr.has('tox-collection__item') ],
+                classes: [ arr.has('tox-collection__item'), arr.has('tox-collection__item--active') ],
```

Comment:
I'll have to update my changes in-line with these changes, but it might be a good idea to assert that the menu does not exist in pOpenMenu to catch the issue with a menu not being closed earlier.

Reply:
Hmm... would we want to assert that though? As the expectation should always be that it's not open. If anything I'd say we should branch to skip opening if it's already open. That said I'm hesitant to make that change though because it seems a little overkill 🤷

Reply:
Actually... on second thoughts I'll make the `closeMenu` function assert it was able to successfully close it. That would make more sense and allows us to keep our expectations.